site/docs/components/StartBuilding.tsx

```diff
@@ -0,0 +1,175 @@
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/identity/address"
+            >
+              addresses
+            </a>
+            .
+          </span>
+        </li>
+        <li className="vocs_ListItem">
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/wallet/wallet"
+            >
+              <code className="vocs_Code">Wallet</code>
+            </a>{' '}
+            - Create or connect wallets with{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/wallet/wallet"
+            >
+              Connect Wallet
+            </a>
+            .
+          </span>
+        </li>
+        <li className="vocs_ListItem">
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/transaction/transaction"
+            >
+              <code className="vocs_Code">Transaction</code>
+            </a>{' '}
+            - Handle{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/transaction/transaction"
+            >
+              transactions
+            </a>{' '}
+            using EOAs or Smart Wallets.
+          </span>
+        </li>
+        <li className="vocs_ListItem">
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/checkout/checkout"
+            >
+              <code className="vocs_Code">Checkout</code>
+            </a>{' '}
+            - Integrate USDC{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/checkout/checkout"
+            >
+              checkout
+            </a>{' '}
+            flows with ease.
+          </span>
+        </li>
+        <li className="vocs_ListItem">
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/fund/fund-button"
+            >
+              <code className="vocs_Code">Fund</code>
+            </a>{' '}
+            - Create a{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/fund/fund-button"
+            >
+              funding
+            </a>{' '}
+            flow to onboard users.
+          </span>
+        </li>
+        <li className="vocs_ListItem">
+          <span>
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
+              href="/token/token-chip"
+            >
+              <code className="vocs_Code">Tokens</code>
+            </a>{' '}
+            - Search and display{' '}
+            <a
+              className="vocs_Anchor vocs_Link vocs_Link_accent_underlined"
```

Comment:
i don't necessarily dislike it, but the hover on these links isn't used anywhere else

Reply:
sorry, i didn't notice that these are currently wrapped in `<strong>` tags. that removes the hover. will update.